apps/passport-server/src/services/telegramService.ts

```diff
@@ -450,12 +450,25 @@ export class TelegramService {
       revealIsRevoked: false
     };
 
+    // For local development, we do not validate eventIds
+    // If you want to test eventId validation locally, copy the `id` field from `pretix_events_config`
+    // and add it to ALLOWED_EVENTS. Then set value: ALLOWED_EVENTS.map((e) => e.eventId)
+    const validEventIds = isLocalServer()
+      ? undefined
+      : ALLOWED_EVENTS.map((e) => e.eventId);
+
     const args: ZKEdDSAEventTicketPCDArgs = {
       ticket: {
         argumentType: ArgumentTypeName.PCD,
         pcdType: EdDSATicketPCDPackage.name,
         value: undefined,
-        userProvided: true
+        userProvided: true,
+        displayName: "Your Tikcet",
+        description: "",
+        validatorParams: {
+          eventIds: validEventIds
```

Comment:
worth adding a `notFoundMessage` here the same way you did in the consumer-client example?